Good-Bids/goodbids
# Auction Status & Auction Start Cron

# Summary

This PR:
* Adds meta data `_auction_started` to new Auctions.
* Sets up a 1min Cron Schedule
* Schedules a Cron to trigger the `goodbids_auction_start_event` action every minute.
* Retrieves any Auctions that have not started with a start date/time <= the current time, with a 1 minute offset.
* Update the `_auction_started` meta

It also:
* Displays the Auction Status for Auctions
* Renames the Reward Product block to Auction Metrics (General) (heads up @nathan-schmidt-viget)
* Cleans up the code a bit
* Adds some basic styling to Auction Info meta box

## Issues

* #26 

## Testing Instructions

This can come later once we have some definitive way of seeing an Auction start. If you want to validate the code, you can create a new Auction and make sure you don't get any errors or run into any issues.

## Screenshots

**Auction Status (Listing View)** - @clatwell Flagging to show how too many columns start to bunch things up.
<img width="1092" alt="Screenshot 2023-12-20 at 2 38 45 PM" src="https://github.com/Good-Bids/goodbids/assets/122309362/4a3769a3-6f20-44c2-9ceb-9c2a989149fa">

**Auction Status (Meta Box)**
<img width="278" alt="Screenshot 2023-12-20 at 2 39 30 PM" src="https://github.com/Good-Bids/goodbids/assets/122309362/e969db35-c509-4d0a-a595-eb50ad83deba">

### Notes

I'm not sure what I did during my Rebase to screw up this branch, but for some reason It includes Nathan's VS Code Settings updates. 🤷
